realm/realm-library/src/main/java/io/realm/internal/LinkView.java

```diff
@@ -153,6 +153,12 @@ public void removeAllTargetRows() {
         nativeRemoveAllTargetRows(nativePointer);
     }
 
+    public Table getTargetTable() {
```

Comment:
It is only a new java `Table` object which points to the same `Table` pointer in the core. No need to cache like other tables we created in java.